docs/esp8266/quickref.rst

```diff
@@ -149,7 +149,7 @@ ADC (analog to digital conversion)
 ADC is available on a dedicated pin.
 Note that input voltages on the ADC pin must be between 0v and 1.0v.
 
-Use the ``machine.ADC`` class::
+Use the :ref:`machine.ADC <machine.ADC>` class::
```

Comment:
Without the `<alias>`, it's overly verbose pattern "class name - description".
I went with the same pattern that was used on the pyboard and wipy docs.

**Test 1** - works, but it's the full class name and description:
```
Use the :ref:`machine.ADC` class::
```
Use the [class ADC – analog to digital conversion]() class:

**Test 2** - works, class name is short and easy to read:
```
Use the :ref:`machine.ADC <machine.ADC>` class::
```
Use the [machine.ADC]() class: